### PR TITLE
DRX Overview

### PR DESCRIPTION
**Pull request by jpgomes**
New Accelerator on DRX

### CODE REVIEW

**Review by alevan10 — CHANGES_REQUESTED** [review]
Just one change, please, and I think it looks good!

File: advanced-experimentation/DRX/drx-overview.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title: Introduction to DataRobotX
```
[the remark this answers is not in the document]
Comment (jpgomes) [reply advanced-experimentation/DRX/drx-overview.yaml]: @alevan10 done. Let me know if this is good to merge now. Thanks

[the remark this answers is not in the document]
Comment (alevan10) [reply advanced-experimentation/DRX/drx-overview.yaml]: looks like it's `.config.yaml.yaml` currently.

[the remark this answers is not in the document]
Comment (jpgomes) [reply advanced-experimentation/DRX/drx-overview.yaml]: should be good now.

**Review by alevan10 — APPROVED** [review]
Thanks for making that change!

**Pull request merged**